From: =?utf-8?q?Sebastian_Dr=C3=B6ge?= <slomo@debian.org>
Date: Wed, 23 Aug 2017 17:12:56 +0300
Subject: 060_pdf_priority

---
 freedesktop.org.xml.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/freedesktop.org.xml.in b/freedesktop.org.xml.in
index 46dbfbd..0f4c13a 100644
--- a/freedesktop.org.xml.in
+++ b/freedesktop.org.xml.in
@@ -185,7 +185,7 @@ command to generate the output files.
     <acronym>PDF</acronym>
     <expanded-acronym>Portable Document Format</expanded-acronym>
     <generic-icon name="x-office-document"/>
-    <magic priority="50">
+    <magic priority="60">
       <match type="string" value="%PDF-" offset="0:1024"/>
     </magic>
     <glob pattern="*.pdf"/>
